src/components/SearchGridManualLoad.vue

```diff
@@ -3,8 +3,10 @@
            ref="searchGrid">
     <div class="search-grid_ctr" ref="gridItems">
       <div v-show="!isFetchingImages && includeAnalytics" class="search-grid_analytics" >
+         <div class="count">
```

Comment:
Is there any reason for adding this div specifically to define styling for the two child elements?

Reply:
okay i will make the changes @akmadian 
Thank you